Find a functionally equivalent code:android.content.RestrictionEntry.getType:COMMENT
Method Modifier: public      
Comment:/**
 * Returns the type for this restriction.
 * @return the type for this restriction
 */

@@ -1,3 +1,3 @@
 {
-    return type;
+    return mType;
 }

